### ISSUE TITLE
Catastrophe 

### ISSUE THREAD

**Issue by KekatD** [siggame/MegaMinerAI-Game-Ideas#38]
Submitted by: Michaela Davis
This is my first time submitting something like this, and I wrote it up in like an hour so there's bound to be some holes in it/stuff that needs to be added.
Updated 9/12/2017

**Story**
Civilization has fallen and the humans have scattered. You are not the savior. Not the last human. Not even weird Uncle Joe who lives in the abandoned mine shaft. You are a cat. Your sole purpose is to gain control over what's left of humanity to do your bidding and take your rightful place as ruler of the world. However, another cat has dared to challenge your right to rule and is amassing their own army of loyal humans. This will not stand. 

**Goal**
Your original following of 5 humans begins spreading the word of your glory to persuade more humans to join your side, and maybe even covert a few away from the enemy. However, you must also make sure you protect yourself from opposing forces and that you have at least the basic necessities. These include food, water, and shelter. 

**Gameplay**
As the supreme being, the cats do nothing besides issue commands from the safety of one of their shelters. The only input involving cats is moving them to a different shelter, which temporarily leaves them open for attack during transit. These shelters must be maintained because they will deteriorate over time. There is a max of 365 turns. At least one human must be devoted to obtaining resources. As your following grows, so does the drain on resources. 

**Units**
_Cat Overlord_: The almighty ruler of the colony. Cats can move to new shelters built by Carpenters, temporarily becoming more vulnerable during transit. Cannot attack enemy units, but can speed up the recovery time of their own units that are in the same shelter that the cat resides in.

_Fresh Human_: This is what the beginning 5 and eventual converted humans start out as.  All humans resting at the shelter will be considered "Fresh Humans" and can therefore change roles whenever at the base.

_Missionary_: These humans have the sole purpose of gaining you new followers. However, they must return to a shelter every 7 days to restock on supplies or when they run out of "Paw Prints" (books detailing your greatness that are given to wild humans to convert them).

_Converter_: A highly skilled missionary who is able to convert the enemy's forces to your side, but much slower than a Missionary and requires more resources. Cannot be obtained for the first 50 days. 

_Thief_: Steals resources from the opponent.

_Gatherer_: Scavenges and forages for food and materials to maintain the almighty cat and the colony. Can only hold up to 10 food and/or materials before needing to return to a shelter.

_Soldier_: Combat units dedicated to either protecting the shelter/cat or attacking the enemy. Can cause damage to shelters and kill non-combat forces. Moves slower and requires more food than the other units. When soldiers clash, the unit with the most energy will win. (Energy meaning how long they've been sent out; soldiers out for a long period of time will become fatigued, fresh soldiers are stronger). If a soldier is part of a squad, their cumulative energy is taken into account. A soldier's energy depletes over time when they are outside the shelter or after they were in combat. Fatigued soldiers are easier to convert to your side.

_Carpenter_: Makes repairs to the shelter and builds/upgrades defenses with "materials" (Walls, trenches, ect.). Can also build new shelters in new locations that allow your humans to go farther now that they have a new resting place or house the cat overlord.

**Resources**
_Wild Humans_: To the cat overlords, humans are resources to be found and retrieved. Considered a stationary resource that can be found at random in neutral territory, either alone or in small groups. 

_Food_: Humans and cats require sustenance to live and be effective at their tasks. Well-fed humans are better and more efficient. Found in water areas (fish) or forests (plants and animals). Over-harvested food areas will yield less food. (Not sure about numbers yet)

_Materials_: Used by the Carpenter(s) for building. Same restrictions as food.

_Paw Prints_: Used by Missionaries to convert wild or enemy humans.

**Victory**
Victory can be obtained in the following conditions:
1.) Breach the enemy's shelter and kill the false ruler.
2.) Your opponent runs out of resources and cannot continue/they have no more humans.
3.) - If the turn limit (365 days/turns) is reached, the cat with the most humans wins. 
     - If they have equal amounts of humans, then the one with the most food wins. 
     - If they have equal food and equal humans, then the one with the most materials wins.
     - If everything is equal, winner is determined by coin flip. (Can probably be expanded more if 
       necessary)

TL;DR (9/12/2017) Updates:
- Expanded victory conditions
- Added "Paw Prints" (used by Missionaries to convert humans)
- Added a few details to resources
- Carpenters can build extra shelters to increase travel distance for humans as well as house the cat overlords
- Cats can travel between shelters, more vulnerable during transit
- Units regain energy/resupply faster if the cat is in the same shelter
- Soldiers become fatigued after combat
- Shelters deteriorate over time
- Removed "At least one human must maintain the shelter every 5 turns" mechanic because pointless
- Humans resting at shelter can switch roles
- Gatherers can only hold up to 10 resources (number can change, just wanted to add a restriction)

Updates mostly credited to input from Jacob Fischer and Jeffrey Strahm
